<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetWork</class>
 <widget class="QWidget" name="NetWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="testBrowser">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tableOut">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图信息输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNodeButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加结点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeInfo">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>详细信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
